src/linker/steps/dev/python_pandas/dummy_step.py

```diff
@@ -25,9 +26,9 @@ def load_file(file_path, file_format=None):
 diagnostics = {}
 
 if "DUMMY_CONTAINER_MAIN_INPUT_FILE_PATHS" in os.environ:
-    main_input_file_paths = os.environ["DUMMY_CONTAINER_MAIN_INPUT_FILE_PATHS"].split(",")
+    main_input_file_paths = json.loads(os.environ["DUMMY_CONTAINER_MAIN_INPUT_FILE_PATHS"])
 else:
-    main_input_file_paths = glob.glob("/input_data/main_input*")
+    main_input_file_paths = glob.glob("/input_data/main_input/*")
```

Comment:
These globs shouldn't really need to change if you are using the environment variables. But it's nbd either way.

Reply:
yeah, i just figured might as well if we're overall moving to routing files using directories instead of appending prefixes to their names

Reply:
Yeah I'd say I'm fine with this change, if you have to rebuild the container anyway.